coding-concepts.js

```diff
@@ -7,53 +7,55 @@
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+//console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: this code will push the string indigo to the end of the existing array
+// b) Verify and explain:The outcome was 5, so this means that i was correct. and the code adds or "pushes" a fifth string to the array. This action gives the new length of 5 instead of 4. This is what the code would have output without the second console command.
 
 
 // --------------------1) What will this log?
 
 const cohort = "Alpha 2022"
-// console.log(cohort.length)
+ //console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: This should come out with, the output of how many characters of text are in the string. This would be the "length" of the phrase in the sting it's output would be 10.
+// b) Verify and explain: I was correct, because it counts the amount of the legnth of the string. computers start counting at 0, so the 9 values in the string are counted as 10. 
 
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+//console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: this will run a greetin that says "Hello world!" 4 times in a row
+// b) Verify and explain: I was wrong, because it's asking for the 4th text value in the string. The output is o which is the 4th letter in the string.
```

Comment:
This one was a tricky one!  It is always good to run problems in a console log to make sure the output is what we expect it to be.